app/models/user.rb

```diff
@@ -490,14 +492,24 @@ def senior_delegate_for?(user)
     user.senior_delegates.include?(self)
   end
 
-  def banned?
-    group_member?(UserGroup.banned_competitors.first)
+  def age_in_years
+    age = Date.today.year - dob.year
+    age -= 1 if Date.today < dob.advance(years: age)
```

Comment:
What?!

Reply:
Restructured to make it easier to grok - originally, this was advancing the dob to the current year to compare the current date to the user's date of birth. New version makes that more explicit